docs/spark-shell.md

```diff
@@ -38,3 +42,8 @@ in either Scala or Python.
 
         $ textFile = sc.textFile("/opt/spark/dist/README.md")
         $ textFile.count()
+
+    In the R shell:
+
+        $ df <- as.DataFrame(faithful)
```

Comment:
"faithful" is a sample dataset that is builtin to R. I took that example from the SparkR doc: http://spark.apache.org/docs/latest/sparkr.html#from-local-data-frames